llvm/test/CodeGen/AMDGPU/GlobalISel/test_g_load_vector_to_scalar.mir

```diff
@@ -0,0 +1,25 @@
+# RUN: not --crash llc -mtriple=amdgcn-mesa-mesa3d -mcpu=tahiti -run-pass=amdgpu-prelegalizer-combiner -verify-machineinstrs %s -o -
```

Comment:
This belongs in test/MachineVerifier, and can use -run-pass=none (and does not need -verify-machineinstrs).

You should also include all the cases in the test I added to the bug (you'll probably need to merge the different MIR instructions into one function to test them in one mir file)

Reply:
Moved and updated, thank you!